### PR TITLE
109 refactor dataingestion classes to use new db schema and object schema

### PR DESCRIPTION
**Pull request by lovelysandlonelys**

### CODE REVIEW

File: src/tests/tests_NOAATANDC.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import pytest
+
+from datetime import datetime, timedelta, time, date
+from DataClasses import Input, Series, TimeDescription, SeriesDescription
+from DataIngestion.IDataIngestion import data_ingestion_factory
+from SeriesStorage.ISeriesStorage import series_storage_factory
+from dotenv import load_dotenv
+
+#datetime.combine(date(2023, 9, 5), time(11, 0))
```
Comment (bestrada33) [inline src/tests/tests_NOAATANDC.py:28]: Is this needed?

File: src/tests/tests_NOAATANDC.py
Comment (bestrada33) [inline src/tests/tests_NOAATANDC.py:1]: Not a HUGE deal, but this test has prefix "tests" while the rest have prefix "test".

File: src/tests/tests_NOAATANDC.py (hunk 1 above)
Comment (matdenkas) [reply src/tests/tests_NOAATANDC.py:28]: 👉👈 That was me, sorry Sav

Comment (lovelysandlonelys) [reply src/tests/tests_NOAATANDC.py:28]: no sir!

File: src/tests/tests_NOAATANDC.py
Comment (lovelysandlonelys) [reply src/tests/tests_NOAATANDC.py:1]: fixed file name!

**Pull request merged**